contrib/sudo-pair.spec

```diff
@@ -0,0 +1,40 @@
+Name: sudo-pair
+Version: v0.9.2
+Release: 1
+Summary: Plugin for sudo that requires another human to approve and monitor privileged sudo sessions.
+Group: tem Environment/Libraries
+License: Apache Software License 2.0
+Url: https://github.com/square/sudo_pair
+Source: https://github.com/square/sudo_pair/archive/sudo_pair-%{version}.tar.gz
+
+BuildRoot: %{_tmppath}/%{name}-%{version}-build
+BuildRequires: cargo
+BuildRequires: clang-devel
+BuildRequires: git
+Requires: sudo
+
+%description
+Plugin for sudo that requires another human to approve and monitor privileged sudo sessions
+
+%global debug_package %{nil}
+
+%prep
+
+%setup -n sudo_pair-sudo_pair-%{version}
+
+%build
+cargo build --release
```

Comment:
Might be worth running `cargo test --release`, just to make sure.

Reply:
Did not know about that. The `test` ensure that it would be able to build or so? In that case; it's also okay if the RPM build process fails with an error. Building an RPM is not done to test the build, but to package the artifact(s).

Reply:
I'll let you decide whatever is conventional here. `cargo build` builds the final artifacts, but `cargo test` makes sure the tests pass in the environment you're building on.